test/cli_run_network_test.go

```diff
@@ -122,3 +122,32 @@ func (suite *PouchRunNetworkSuite) TestRunWithIP(c *check.C) {
 
 	c.Assert(found, check.Equals, true)
 }
+
+// TestRunAddHost is to verify run container with add-host flag
+func (suite *PouchRunNetworkSuite) TestRunAddHost(c *check.C) {
+	res := command.PouchRun("run", "--add-host=extra:86.75.30.9", "busybox", "grep", "extra", "/etc/hosts")
```

Comment:
we need to add a `defer DelContainerForceMultyTime(c, cname)` to clean up, same with other test case.

Reply:
update this？

Reply:
There is no name when running the container, how can I add a defer DelContainerForceMultyTime(c, cname) to clean up it?

Reply:
why not use --name to assign a name to container

Reply:
I'm so sorry that I didn't understand what you meant at first. Thanks for your advice.